### PR TITLE
use correct path for readme

### PR DESCRIPTION
**Pull request by Bucknalla**
Change-type: patch
Signed-off-by: Alex Bucknall <alex.bucknall@gmail.com>

### CODE REVIEW

**Comment by Bucknalla** [comment]
@balena-ci I self-certify!

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,20 +8,23 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
...
+      return process.env.PROJECT_NAME
+    }
   } catch (error) {
     throw error
   }
 }
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: getFrontmatter('project_name') || process.env.PROJECT_NAME || "",
```
Comment (vipulgupta2048) [inline docusaurus.config.js:24]: Hey @Bucknalla can you explain why we are removing the default? I would say we should have something like this instead. This change would be breaking project builds all over the place for those that don't have a frontmatter and title can't be empty

`getFrontmatter('project_name') || process.env.PROJECT_NAME`

Comment (vipulgupta2048) [reply docusaurus.config.js:24]: I see now what you did. But, it makes the function too specific to only be used for the README's frontmatter capable of only returning project_name. Sooner than later we would hit another wall. Making an improvement.

Comment (vipulgupta2048) [reply docusaurus.config.js:24]: Opened #38

**Pull request merged**